doc/kyuafile.5.in

```diff
@@ -290,6 +290,15 @@ it can run.
 .Pp
 ATF:
 .Va require.files
+.It Va required_kmods
+Whitespace-separated list of kernel module names that the test requires to
+be loaded before it can run.
+This requirement checking is platform-dependent.
+It is ignored for a non-supported platform.
+Supported platforms: FreeBSD.
```

Comment:
I suggest using `.Fx` here instead of `FreeBSD`:
```suggestion
Supported platforms:
.Fx
```

Reply:
Indeed, thanks. Fixed.